### DISCUSSION TITLE
Any word on native windows executables?

### DISCUSSION THREAD

**Discussion by heron1** [dotnet/maui#14889] [General]
I'm not sure who the target audience is for .Net MAUI on Windows, but expecting small time developers to purchase trusted keys with yearly fees for their apps or the end user to understand how to install a self-signed key is not practical. 

I'm eager to make the transition to Blazor & .Net MAUI but something as simple having the code compile to native .exe seems still not possible. If this isn't planned to ever be done, can I ask why? And if it is, what's the roadmap? If it's a security issue even languages like python have libraries that compile to .exe (and yes, I'm aware .Net too), but why wouldn't Microsoft offer a way to do so in Blazor/MAUI  which is supposed to be unifying everything?

**Comment by Xyncgas** [comment]
We need this because if one day MAUI is dead like windows phone, or webview is dead because chrome is dead or the web is dead, or the internet or the windows store or microsoft is dead we still need the program to run forever on people's computer

MAUI should support producing Stand alone executable
